docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
+domains in [Question 12].
 
 Lastly, this open source version of Zonemaster was built using modular code
-which, basically, means you can use parts of it in your systems, if you would want
-to. It is quite rare that you'd want a complete program just to check for example
+which basically means that you can integrate parts of it in your own systems, if you wish.
+For example, it is quite rare that you would want a complete program just to check for
 redelegations.
 
 <a name="q9"></a>
 #### 9. Zonemaster and privacy
 Since Zonemaster is open to everyone it is possible for anyone to check your
-domain and also see previous tests, however there is no way to tell
-who has run a specific test since nothing is logged except the time of the test.
+domain and its tests history.
+However there is no way to tell who has run a specific test since nothing is
+logged except the start time of the test.
 
 <a name="q10"></a>
-#### 10. How come I cannot test my domain?
-If we disregard the situation where the domain does not exist, as in when you input a
-non-existing domain to Zonemaster, there are 2 other possibilites:
-  - To protect the engine from multiple identical inputs, that is the same IP
-    checking the same zone several times, there is a delay of 5 minutes between
-    identical subsequent tests. This means that you can only test the
-    same domain once every 5 minutes, if you try and test it again within 5 minutes
-    the last results will be displayed instead.
-  - Because Zonemaster was made to check domains or DNS zones, like zonemaster.net, and
-    not hostnames in a domain (zone), like www.zonemaster.net, the Zonemaster will
-    report a failure if you try to test a host name instead of a domain matching a
+#### 10. How come my domain cannot be tested?
+If we disregard the situation where a domain does not exist, such as when a
+non-existing domain is input to Zonemaster, there are 2 other possibilities:
+  - To protect the Engine from repetitive inputs, being the same IP
+    checking the same domain several times, there is a delay of 5 minutes between
+    subsequently identical tests.
+    This means that you can only test the same domain once every 5 minutes,
+    and if you try and test it again during that time the last results
+    will be displayed instead.
+  - Because Zonemaster was made to check domains, or DNS zones (e.g. 'zonemaster.net'), and
+    not hostnames in a domain (e.g. 'www.zonemaster.net'), Zonemaster will
+    report a failure if you try to test a hostname instead of a domain matching a
     DNS zone.
 
 <a name="q11"></a>
 #### 11. What kind of queries does Zonemaster generate?
-Zonemaster send multiple DNS queries to the name servers hosting the domain name and
-also to the name servers hosting the parent zone of the domain name.
+Zonemaster send multiple DNS queries to the name servers hosting the domain tested and
+also to the name servers hosting the parent zone of the domain tested.
```

Comment:
“Domain tested” is better written as “domain being tested”. Or maybe “domain under test”?

Reply:
Updated.